metric/system/process/helpers.go

```diff
@@ -98,3 +100,10 @@ func GetProcCPUPercentage(s0, s1 ProcState) ProcState {
 	return s1
 
 }
+
+func CanIgnore(err error) bool {
```

Comment:
Does this need to be exported, as it seems to be used from within the same package?
```suggestion
func canIgnore(err error) bool {
```

Same goes for `CanDegrade`.

Reply:
Hi, 
Regarding `CanDegrade`, we use this helper in metricbeat [here](https://github.com/elastic/beats/pull/40298).

Reply:
Also, @ycombinator we will also use `CanDegrade` [here](https://github.com/elastic/elastic-agent/blob/883c6867e62d1b1eec3534568dc8c7c76bbb8ced/pkg/utils/watcher.go#L26-L29) to ignore non-fatal errors.

Reply:
I've made `CanIgnore` local to this package.